resources/nl-core/nl-core-HelpFromOthers.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="nl-core-HelpFromOthers" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/nl-core-HelpFromOthers" />
+  <name value="NlcoreHelpFromOthers" />
+  <title value="nl core HelpFromOthers" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="url" />
+      <value value="https://www.nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="There are often multiple people or parties involved in the care for a patient with a disorder or disability, particularly in the event of home care. Their efforts enable the patient to function more or less independently. This includes not only caregivers, but also professional help such as that offered by home care organizations, nurses, aids and helpers.&#xD;&#xA;Current doctors and staff from the facility to which a patient is admitted do not fall under this concept." />
+  <purpose value="A derived profile from [zib-HelpFromOthers](http://nictiz.nl/fhir/StructureDefinition/zib-HelpFromOthers) to provide a version better suited for implementation purposes. This profile augments the base profile with elements found in the various use cases that have adopted the zib." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <kind value="resource" />
+  <abstract value="false" />
+  <type value="CarePlan" />
+  <baseDefinition value="http://nictiz.nl/fhir/StructureDefinition/zib-HelpFromOthers" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="CarePlan">
+      <path value="CarePlan" />
+      <alias value="nl-core-HelpFromOthers" />
+    </element>
```

Comment:
Add reference to nl-core-Patient on `.subject`.

Reply:
Fix